internal/procinfo/pidtouser.go

```diff
@@ -0,0 +1,51 @@
+package procinfo
+
+import (
+	"errors"
+	"github.com/gpuctl/gpuctl/internal/passwd"
+	"os"
+	"strconv"
+	"syscall"
+)
+
+var (
+	ErrorNoUserFound = errors.New("Could not find user that owns the given process from passwd file")
+)
+
+type UidLookup map[uint32]string
+
+func (lookup UidLookup) Get(pid uint64) (string, error) {
+	var zero string
```

Comment:
NIT: Given that `""` is literally twice as short as `zero`, I'd rather we used that.